catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -328,6 +328,11 @@ private void commitUpdates(StorageRequest request) throws StorageException {
       for (String contentUri : updateMap.get(request.getId())) {
         Path contentIdDir = getTempContentItemDir(request.getId(), new URI(contentUri));
         Path target = getContentItemDir(new URI(contentUri));
+        if (target == null) {
+          throw new StorageException(
```

Comment:
❓ Is throwing an exception here changing the semantics of what we do when a single item fails out of the collection of items being updated?

Reply:
Changed if body to log the condition and then continue with the next item